centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -21,10 +21,7 @@
 import com.google.gson.JsonObject;
 import com.google.protobuf.InvalidProtocolBufferException;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Lets stay specific in what we are importing.

Reply:
ok